### DISCUSSION TITLE
"Script Host Kick" unblockable for us?

### DISCUSSION THREAD

**Discussion by snowleopardd** [YimMenu/YimMenu#3149] [Q&A]
So after finding the "unblockable" kick method this guy used against me on https://github.com/YimMenu/YimMenu/issues/3119 i began  testing it on other modders, but our method seems to not work at all against modders, in that issue we can see that neither both of us were host, but when i try to use the same kick method against other modders without host it just gets blocked and detected by them, what's the deal?

I'm aware of the "freemode owner" thing, i already tried using the "script host" but it still doesn't do anything (not talking about being the genuine host)...

**Comment by lonelybud** [comment]
Script Host Kick works but it wont work on one with Script Host Kick protection. I would rather say not to use it since it can be detected and is "Blocked by most menus"

**Reply by snowleopardd** [reply]
so we're the ones in a disadvantage? we dont have that kick protection

**Comment by Rxann** [comment]
> so we're the ones in a disadvantage? we dont have that kick protection

Yes we do

**Reply by snowleopardd** [reply]
but the protection only works if we are host right?

**Comment by maybegreat48** [comment]
"SH Kick" is likely the desync kick, not a script host kick. If so, yes, it's only truly blockable when host

**Reply by lonelybud** [reply]
so the nat_type = 0 stuff is only added to protect against users who do not have nat_type = 0.

It must be like there are more parameters to take on for example having small host token

**Reply by Rxann** [reply]
> so the nat_type = 0 stuff is only added to protect against users who do not have nat_type = 0.
> 
> 
> 
> It must be like there are more parameters to take on for example having small host token

that is to deflect the desync kick by setting the NAT type to unknown.
